py3status/modules/volume_status.py

```diff
@@ -8,12 +8,9 @@
 example.
 
 Configuration parameters:
-    button_down: Button to click to decrease volume. Setting to 0 disables.
-        (default 0)
-    button_mute: Button to click to toggle mute. Setting to 0 disables.
-        (default 0)
-    button_up: Button to click to increase volume. Setting to 0 disables.
-        (default 0)
+    button_down: mouse button to decrease volume (default None)
+    button_mute: mouse button to toggle mute (default None)
+    button_up: mouse button to increase volume (default None)
```

Comment:
I'd prefer if these where just `button_down: button to decrease volume (default None)` because they do not need to be a mouse button.

Yes some modules use do say mouse button but such is life.

Other than that this looks good

Reply:
`click event`? Also, what about changing `None,None` to `4,5`? We leave `mute` alone.

This is consistent with i3's default bar for workspace scrolling.
```
# disable workspace scrolling
bar {
   bindsym button4 nop
   bindsym button5 nop
}
```
Reason we don't want `mute` is because `button_down` works just fine. Keep it simple.
And everybody who uses this module wouldn't have to keep/configure anymore.

Reply:
> click event? 

No, the whole point is to try an have consistency/uniformity.  click event sounds terrible and is nothing to do with the user experience.

> Also, what about changing None,None to 4,5? 

No thanks.  Can we try to keep things really simple.

Reply:
Yuh. Updated. Let's merge/close this PR.